html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-001.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML: window.open `features`: tokenization -- `noopener`</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#apis-for-creating-and-navigating-browsing-contexts-by-name">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var windowURL = 'resources/close-self.html';
+
+// Tests for how windows features are tokenized into 'name', 'value'
+// window features separators are ASCII whitespace, '=' and  ','
+
+test (t => {
+  // Tokenizing `name`: initial window features separators are ignored
+  // Each of these variants should tokenize to ('noopener', '')
+  var featureVariants = [
+    ' noopener',
+    '=noopener',
+    ',,noopener',
+    ',=, noopener',
+    '\n=noopener=',
+    '\tnoopener',
+    '\r,,,=noopener',
+    '\u0000noopener',
```

Comment:
Wait, this one is wrong. U+0000 is not a separator; `('\0noopener', '')` is not a supported feature.